samples/cpp/benchmark_app/main.cpp

```diff
@@ -424,28 +427,56 @@ int main(int argc, char* argv[]) {
             next_step();
             // Parse input shapes if specified
             bool reshape = false;
-            app_inputs_info = getInputsInfo<InputInfo::Ptr>(FLAGS_shape,
-                                                            FLAGS_layout,
-                                                            FLAGS_b,
-                                                            FLAGS_data_shape,
-                                                            FLAGS_iscale,
-                                                            FLAGS_imean,
-                                                            inputInfo,
-                                                            reshape);
+            app_inputs_info = getInputsInfo(FLAGS_shape,
```

Comment:
get_inputs?

Reply:
nope, it actually returns inputs_info, so, let's keep this name